README.md

```diff
@@ -1,16 +1,26 @@
 # Harmony
-[![Build Status](https://travis-ci.com/harmony-one/harmony.svg?token=DnoYvYiTAk7pqTo9XsTi&branch=master)](https://travis-ci.com/harmony-one/harmony)
+[![Build Status](https://travis-ci.org/harmony-one/harmony.svg?branch=master)](https://travis-ci.org/harmony-one/harmony)
```

Comment:
Nice catch (and mea culpa; I was the one who switched Harmony to travis-ci.org).  :-)

Reply:
Maybe it is time to switch back - https://docs.travis-ci.com/user/migrate/open-source-on-travis-ci-com/ :)